assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -309,6 +309,9 @@ che.infra.kubernetes.namespace.default=<username>-che
 # Defines whether che-server should try to label the workspace namespaces.
 che.infra.kubernetes.namespace.label=true
 
+# Defines whether che-server should try to annotate the workspace namespaces.
+che.infra.kubernetes.namespace.annotate=true
```

Comment:
why do we need this flag? why don't we act as we do with labels?

Reply:
Maybee somebody should only create label or annotation, but not the two.
If we should only one flag to drive or not label and annotation, maybee the flag `che.infra.kubernetes.namespace.label` must be renamed ?

Reply:
I would expect some kind of complementary behavior. don't you agree?

Reply:
Let's rename it to `che.infra.kubernetes.namespace.annotation` to be consistent with labels.

Reply:
it's done.